### DISCUSSION TITLE
请问 MFSU 运行警告怎么处理？

### DISCUSSION THREAD

**Discussion by 973001046** [umijs/umi#11112] [Q&A] [answered]
[MFSU] require('/Users/raojun/code/FrameworkUpdate/src/locales/zh-CN.ts') found in /Users/raojun/code/FrameworkUpdate/src/.umi/plugin-locale/localeExports.ts, which will broken the MFSU prebuild match, please change to esm module import.
[MFSU] require('antd/es/locale/zh_CN') found in /Users/raojun/code/FrameworkUpdate/src/.umi/plugin-locale/localeExports.ts, which will broken the MFSU prebuild match, please change to esm module import.

**Comment by fz6m** [comment, marked as answer]
你的 `src/locales/zh-CN.ts` 这个文件可能写法有问题，正常的写法是 `export default { ... }` ，如有问题，需要给一个最小复现。

**Reply by 973001046** [reply]
<img width="639" alt="image" src="https://user-images.githubusercontent.com/18065985/236819372-40f36e9f-3856-4e76-9d3b-f702aa854c54.png"> 我是这样配置
